src/app/settings/views/Configuration/GeneralForm/GeneralForm.test.tsx

```diff
@@ -112,10 +165,16 @@ describe("GeneralForm", () => {
         </MemoryRouter>
       </Provider>
     );
-    submitFormikForm(wrapper, {
-      enable_analytics: true,
-      release_notifications: false,
+
+    const release_notifications_checkbox = screen.getByRole("checkbox", {
+      name: "Enable new release notifications",
     });
+
+    const saveButton = screen.getByRole("button", { name: "Save" });
+
+    await userEvent.click(release_notifications_checkbox);
```

Comment:
Is there a reason why why you decided not to stick to a single naming convention (camelCase in `saveButton` vs snake_case in `release_notifications_checkbox`)?